packages/lodestar/test/sim/merge-interop.test.ts

```diff
@@ -46,7 +46,7 @@ import {bytesToData, dataToBytes, quantityToNum} from "../../src/eth1/provider/u
 // 10 ttd / 2 difficulty per block = 5 blocks * 5 sec = 25 sec
 const terminalTotalDifficultyPreMerge = 10;
 const TX_SCENARIOS = process.env.TX_SCENARIOS?.split(",") || [];
-const jwtSecretHex = "dc6457099f127cf0bac78de8b297df04951281909db4f58b43def7c7151e765d";
+const jwtSecret = "dc6457099f127cf0bac78de8b297df04951281909db4f58b43def7c7151e765d";
```

Comment:
Declare here with 0x prefix

Reply:
fixed in https://github.com/ChainSafe/lodestar/pull/3777/commits/46e723b0912b42392276d384059d5aadbcb0cc43